src/abstractdataframe/abstractdataframe.jl

```diff
@@ -365,39 +371,57 @@ If the column does not allow missing values, `nothing` is returned.
 Consequently, `nmissing = 0` indicates that the column allows
 missing values, but does not currently contain any.
 
+If custom functions are provided, they are called repeatedly with the vector corresponding
+to each column as the only argument. For columns allowing for missing values,
+the vector is wrapped in a call to [`skipmissing`](@ref): custom functions must therefore
+support such objects (and not only vectors), and cannot access missing values.
+
+
 **Examples**
 
 ```julia
 df = DataFrame(i = 1:10, x = rand(10), y = rand(["a", "b", "c"], 10))
 describe(df)
-describe(df, stats = :all)
-describe(df, stats = [:min, :max])
+describe(df, :all)
+describe(df, :min, :max)
+describe(df, :min, :sum => sum)
 ```
 
 """
-function StatsBase.describe(df::AbstractDataFrame; stats::Union{Symbol,AbstractVector{Symbol}} =
-                            [:mean, :min, :median, :max, :nunique, :nmissing, :eltype])
-    # Check that people don't specify the wrong fields.
+StatsBase.describe(df::AbstractDataFrame) = _describe(df, [:mean, :min, :median, 
```

Comment:
Better put a line break here so that the array is on a single line. That will also fix the indentation below.

Reply:
fixed.